dictionaries/nl.dictionary.itop.ui.php

```diff
@@ -786,7 +793,7 @@
 	'UI:Schema:Attribute/Filter' => 'Filter',
 	'UI:Schema:DefaultNullValue' => 'Standaardwaarde null : "%1$s"',
 	'UI:LinksWidget:Autocomplete+' => 'Typ de eerste 3 karakters...',
-	'UI:Edit:SearchQuery' => 'Select a predefined query~~',
+	'UI:Edit:SearchQuery' => 'Kies een voorgedefinieerde query',
```

Comment:
Might sound clearer:
```suggestion
	'UI:Edit:SearchQuery' => 'Kies een vooraf gedefinieerde query',
```

Reply:
Solved